add snapshot instruction for Testnet (#1001)
filecoin-project/filecoin-docs: docs/get-started/lotus/chain.md

1. Download the most recent lightweight snapshot and its checksum:

    **Mainnet**

    + [This URL](https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car) always contains the latest snapshot available for mainnet.

    ```shell
    curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car | perl -ne '/x-amz-website-redirect-location:\s(.+)\.car/ && print "$1.sha256sum\n$1.car"' | xargs wget
    ```

>     **TestNet**

Reviewer: ```suggestion
    **Testnet**
```

>     + Calibration: snapthot is maintained by Filecoin community. Download latest snapshot: [lotus_cali_snapshot_2021_07_14_high_73770.car.tar.xz](https://www.mediafire.com/file/q7tc2bmcc9d09vv/lotus_cali_snapshot_2021_07_14_high_73770.car.tar.xz/file). 

Reviewer: ```suggestion
    + Calibration: snapshot is maintained by Filecoin community. Download latest snapshot: [lotus_cali_snapshot_2021_07_14_high_73770.car.tar.xz](https://www.mediafire.com/file/q7tc2bmcc9d09vv/lotus_cali_snapshot_2021_07_14_high_73770.car.tar.xz/file). 
```

    + Nerpa: also maintained by Filecoin Community.

>       + Option-1: download latest snapshot [lotus_nerpa_snapshot_2021_07_14_high_339089.car.tar.xz](https://www.mediafire.com/file/t1j360oyk3tjziv/lotus_nerpa_snapshot_2021_07_14_high_339089.car.tar.xz/file)

Reviewer: ```suggestion
      + Option 1: Download the latest static snapshot [lotus_nerpa_snapshot_2021_07_14_high_339089.car.tar.xz](https://www.mediafire.com/file/t1j360oyk3tjziv/lotus_nerpa_snapshot_2021_07_14_high_339089.car.tar.xz/file)
```

>       + Option-2: this snapshot is taken with the latest 900 blocks daily which is around 1.2GB now. You can directly import snapshop using:

Reviewer: ```suggestion
      + Option 2: Auto-generated snapshot, taken with the latest 900 blocks daily. Size is ~1.2GB as of block height 1,000,000 (August 2021). To import this snapshot directly:
```

1. Download the most recent lightweight snapshot and its checksum:

    **Mainnet**

    + [This URL](https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car) always contains the latest snapshot available for mainnet.

    ```shell
    curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car | perl -ne '/x-amz-website-redirect-location:\s(.+)\.car/ && print "$1.sha256sum\n$1.car"' | xargs wget
    ```

    **TestNet**

    + Calibration: snapthot is maintained by Filecoin community. Download latest snapshot: [lotus_cali_snapshot_2021_07_14_high_73770.car.tar.xz](https://www.mediafire.com/file/q7tc2bmcc9d09vv/lotus_cali_snapshot_2021_07_14_high_73770.car.tar.xz/file). 

    + Nerpa: also maintained by Filecoin Community.

      + Option-1: download latest snapshot [lotus_nerpa_snapshot_2021_07_14_high_339089.car.tar.xz](https://www.mediafire.com/file/t1j360oyk3tjziv/lotus_nerpa_snapshot_2021_07_14_high_339089.car.tar.xz/file)

      + Option-2: this snapshot is taken with the latest 900 blocks daily which is around 1.2GB now. You can directly import snapshop using:

        ```shell

>         lotus daemon --halt-after-import --import-snapshot https://dev.node.glif.io/nerpa00/ipfs/8080/ipfs/$(curl -s https://gist.githubusercontent.com/openworklabbot/d32543d42ed318f6dfde516c3d8668a0/raw/snapshot.log) 
>         or
>         lotus daemon --halt-after-import --import-snapshot https://gateway.ipfs.io/ipfs/$(curl -s https://gist.githubusercontent.com/openworklabbot/d32543d42ed318f6dfde516c3d8668a0/raw/snapshot.log)

Reviewer 2: A gateway is a gateway, no need to point to specific URLs

```suggestion
        lotus daemon --halt-after-import --import-snapshot https://dweb.link/ipfs/$(curl -s https://gist.githubusercontent.com/openworklabbot/d32543d42ed318f6dfde516c3d8668a0/raw/snapshot.log) 
```

Author: I think we can also download file on IPFS via IPFS gateway, right ?

Reviewer 2: This is what my change does: the canonical gateway to use for all IPFS links is `https://dweb.link/ipfs/...`

Author: Got you. Thank you for the suggestion.
